emu/src/memory/internal_memory.rs

```diff
@@ -29,8 +33,9 @@ pub struct InternalMemory {
     /// From 0x06000000 to 0x06017FFF (96 kb).
     pub video_ram: Vec<u8>,
 
-    /// From 0x04000100 to 0x0400010E.
-    timer_registers: TimerRegisters,
+    /// From 0x08000000 to 0x0FFFFFFF.
+    /// Basically here you can find different kind of rom loaded.
+    pub rom: Arc<Mutex<Vec<u8>>>,
```

Comment:
Do we need the rom to be wrapped around `Arc<Mutex<>>` ?

Reply:
Don't think we need now, I think it was like this because of some "viewer" to create in UI then should be shared

Reply:
But in case I think that from the viewer we have the gba/cpu and inside that the InternalMemory is already under `Arc<Mutex<>>` so maybe for now we can remove it from the rom and see if in future we need it

Reply:
Done